build-tools/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -172,7 +172,7 @@
         </module>
         <module name="AbbreviationAsWordInName">
             <property name="ignoreFinal" value="false"/>
-            <property name="allowedAbbreviationLength" value="1"/>
```

Comment:
Do we have longer counter examples than `DB` followed by the capital for the next word? A setting of `2` captures that case.